### ISSUE TITLE
Add publisher/journal: Royal Entomological Society

### ISSUE THREAD

**Issue by mariaguerreiro** [datadryad/dryad-product-roadmap#3386]
**Name to appear on membership page:** 
Royal Entomological Society

**Payment plan:**
Annual Membership + Quarter DPCs (handled via Wiley)

**Journals included:** 
Agricultural and Forest Entomology
Ecological Entomology
Insect Conservation and Diversity
Insect Molecular Biology
Medical and Veterinary Entomology
Physiological Entomology
Systematic Entomology

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)** 
See spreadsheet: https://docs.google.com/spreadsheets/d/1qW15Lly7MifltFTqlfBFcjbPGlMhe2I6M31Rral38Ao/edit?usp=sharing

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)**
See spreadsheet: https://docs.google.com/spreadsheets/d/1qW15Lly7MifltFTqlfBFcjbPGlMhe2I6M31Rral38Ao/edit?usp=sharing 

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)**
Emilie​​​​ Aimé, ORCID: 0000-0003-2290-584X 

**Email contacts to discuss integration of journal metadata with Dryad:**
See spreadsheet: https://docs.google.com/spreadsheets/d/1qW15Lly7MifltFTqlfBFcjbPGlMhe2I6M31Rral38Ao/edit?usp=sharing

Development tasks:
- [ ] add member name to membership page
- [x] update payment settings
- [x] update journal notification settings
- [x] add journal administrators
- [x] start integration process (track on new ticket)

cc @jleighherzog
